web/Web/web/core/models.py

```diff
@@ -42,16 +42,17 @@ class Session(models.Model):
                                       editable=False)
     updated_at = models.DateTimeField(auto_now=True)
 
-    def save(self, *args, **kwargs):
-        if not self.pk:
-            try:
-                CALFunctions.add_session(str(self.uuid),
-                                         self.topic.seed_query,
-                                         self.strategy)
-            except (CALError, ConnectionRefusedError, Exception) as e:
-                # TODO: log error
-                pass
-        super(Session, self).save(*args, **kwargs)
+    def begin_session_in_cal(self):
+        try:
+            judgments = self.judgment_set.all()
```

Comment:
I haven't seen `self.judgment_set` before. What is it/what does it do?

Reply:
For one to many relationships you can access children using `parent.child_set`.